VARIABLES.md

```diff
@@ -958,15 +1006,15 @@ Default:  "{{ monitoring_interceptors_enabled }}"
 
 ### control_center_user
 
-Only use to customize Linux User Control Center Service runs with. User must exist on host.
+Only use to customize Linux User Control Center Service runs with. Default user is cp-control-center
 
 Default:  "{{control_center_default_user}}"
 
 ***
 
 ### control_center_group
 
-Only use to customize Linux Group Control Center Service user belongs to. Group must exist on host.
+Only use to customize Linux Group Control Center Service user belongs to. Default group is confluent
```

Comment:
Modify this variable to customize the Linux Group that the Control Center Service user belongs to. Default group is confluent.